session/auth/authprovider/tokenseed.go

```diff
@@ -69,7 +69,7 @@ func (ts *tokenSeeds) getSeed(host string) ([]byte, error) {
 	}
 
 	if err := ioutil.WriteFile(fp, dt, 0600); err != nil {
-		if !errors.Is(err, syscall.EROFS) && !errors.Is(err, syscall.EPERM) {
+		if !errors.Is(err, syscall.EROFS) && !errors.Is(err, syscall.EPERM) && !errors.Is(err, syscall.EPERM) && !errors.Is(err, syscall.EACCES) {
```

Comment:
You are checking for `EPERM` twice here.
Also, you should do `!errors.Is(err, syscall.EROFS) && !errors.Is(err, os.ErrPermission)` for simplicity instead, as suggested by the standard library.  Same for the other checks.